docker-compose/docker-compose.test.postgres.yml

```diff
@@ -0,0 +1,119 @@
+      - Modules__Messages__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=messages;Password=Passw0rd;Port=5432
+      - Modules__Messages__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Relationships__Infrastructure__BlobStorage__CloudProvider=Azure
+      - Modules__Relationships__Infrastructure__BlobStorage__ConnectionInfo=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
+      - Modules__Relationships__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=relationships;Password=Passw0rd;Port=5432
+      - Modules__Relationships__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Synchronization__Infrastructure__BlobStorage__CloudProvider=Azure
+      - Modules__Synchronization__Infrastructure__BlobStorage__ConnectionInfo=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
+      - Modules__Synchronization__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=synchronization;Password=Passw0rd;Port=5432
+      - Modules__Synchronization__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Tokens__Infrastructure__BlobStorage__CloudProvider=Azure
+      - Modules__Tokens__Infrastructure__BlobStorage__ConnectionInfo=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;
+      - Modules__Tokens__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=tokens;Password=Passw0rd;Port=5432
+      - Modules__Tokens__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Devices__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Devices__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=devices;Password=Passw0rd;Port=5432
+
+      - Modules__Challenges__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Challenges__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=challenges;Password=Passw0rd;Port=5432
+
+      - Modules__Quotas__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Quotas__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=quotas;Password=Passw0rd;Port=5432
+
```

Comment:
I did consider it. Is that preferred? Our currently existing `appsettings.override.json` file currently sets up more than the database. We'd have all those settings duplicated.

Reply:
There shouldn't be THAT much more. It's mostly the pagination stuff. But I think it would make sense to move the pagination settings to the appsettings.json file to have it as a default anyway. Just like the `Modules.Devices.ApplicationOptions.AddressPrefix`.

Reply:
How about the BlobStorage Settings?
And the other settings, such as EventBus and Serilog? Should all these be moved to the `appsettings.json` file as well?

Reply:
Right, I forgot about those. They don't make sense as default values, of course. You can duplicate them.